app/assets/stylesheets/bootstrap_and_overrides.css.scss

```diff
@@ -0,0 +1,13 @@
+.bootstrap {
```

Comment:
I don't care for any of these global overrides. I think we'll be best off if we stay close to vanilla Bootstrap until we're 100% migrated.

If they were to remedy conflicts with Tracks styles, check out #1924.